avalanche/training/supervised/ar1.py

```diff
@@ -229,7 +229,9 @@ def _before_training_exp(self, **kwargs):
             ]
             self.cwr_plugin.reset_weights(self.cwr_plugin.cur_class)
 
-    def make_train_dataloader(self, num_workers=0, shuffle=True, **kwargs):
+    def make_train_dataloader(
+        self, num_workers=0, shuffle=True, persistent_workers=True, **kwargs
```

Comment:
I'm not a  big fan of overriding default pytorch dataloader arguments. `persistent_workers` is `False` by default.

Reply:
I'll revert it to False